testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -2727,6 +2727,7 @@ QUnit.test("removing tag by backspace should not load data from DS", function(as
             }
         },
         value: ["one", "two"],
+        deferRendering: true,
```

Comment:
Why do you change this test?

Reply:
DataSource.load method calls 2 times: one call for loadTagData and one call for inner dxList. Earlier there was only one call: for the inner list. Now this test check only one call: for the `loadTagData` method. I have enabled deferRendering to prevent unexpected call of `load` method